.github/actions/smoke-test/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+TEMPLATE_ID="$1"
+
+set -e
+
+shopt -s dotglob
+
+SRC_DIR="/tmp/${TEMPLATE_ID}"
+cp -R "src/${TEMPLATE_ID}" "${SRC_DIR}"
+
+pushd "${SRC_DIR}"
+
+# Configure templates only if `devcontainer-template.json` contains the `options` property.
+OPTION_PROPERTY=( $(jq -r '.options' devcontainer-template.json) )
```

Comment:
Is OPTION_PROPERTY intended to be an array? From the downstream usage, it doesn't look like it. If not, then remove the outer ().

Wrap the subshell in double quotes:

```
readonly OPTION_PROPERTY="$(jq -r '.options' devcontainer-template.json)"
```

https://google.github.io/styleguide/shellguide.html#quoting